app/containers/ItunesContainer/TrackDetail/index.js

```diff
@@ -0,0 +1,85 @@
+/**
+ *
+ * TrackDetail
+ *
+ */
+
+import React, { useEffect, memo } from 'react';
+import { useParams } from 'react-router';
+import PropTypes from 'prop-types';
+import { injectIntl } from 'react-intl';
+import styled from 'styled-components';
+import { createStructuredSelector } from 'reselect';
+import { isEmpty } from 'lodash';
+import { compose } from 'redux';
+import { injectSaga } from 'redux-injectors';
+import { connect } from 'react-redux';
+import { Skeleton } from 'antd';
+import { itunesContainerCreators } from '../reducer';
+import { selectItunesContainer, selectTrackData, selectTrackError } from '../selectors';
+import { trackDetailsSaga } from '../saga';
+import TrackCard from '@app/components/TrackCard';
+import If from '@app/components/If';
+import { T } from '@app/components/T';
+
+const Container = styled.div`
+  display: flex;
+  flex-direction: column;
+  width: 100%;
+  margin: 0 auto;
+`;
+
+export function TrackDetail({
+  dispatchTrackData,
+  dispatchClearTrackData,
+  intl,
+  trackDetails,
+  trackError,
+  maxwidth,
+  padding
+}) {
+  const { trackId } = useParams();
+  useEffect(() => {
+    dispatchClearTrackData();
+    dispatchTrackData(trackId);
+  }, []);
+  return (
+    <Container>
+      <If condition={isEmpty(trackError)} otherwise={<T id="something_went_wrong" />}>
```

Comment:
Definitely work on adding more UI elements. Go through the antd documentation and use some of the components.